### PR TITLE
Fixed spec tests and default recipe to handle new st2auth config

### PR DESCRIPTION
**Pull request by djolly-fuze**
Fixed the spec and recipes as requested in #74

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # stackstorm cookbook CHANGELOG
 
 ## In Development
+## 0.7.1
```
Comment (arm4b) [inline CHANGELOG.md]: Let's please place this fix under the `v0.7.0` changelog as I didn't release new version after the most recent PR #74

File: metadata.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 license 'Apache-2.0'
 description 'Installs/Configures stackstorm'
 long_description 'Installs/Configures stackstorm'
-version '0.7.0'
+version '0.7.1'
```
Comment (arm4b) [inline metadata.rb]:
```suggestion
version '0.7.0'
```
per previous comment

**Review by arm4b — CHANGES_REQUESTED** [review]
Thanks for the follow-up to #74!

There are still some minor test inconsistencies to fix, according to CI job https://travis-ci.org/StackStorm/chef-stackstorm/jobs/623249551

File: CHANGELOG.md (hunk 1 above)
Comment (djolly-fuze) [reply CHANGELOG.md]: Sure thing!

**Comment by djolly-fuze** [comment]
Ok, I *think* it's good now.
Sorry for the trouble, not used to writing spec tests.

**Comment by arm4b** [comment]
That's good, thanks for the fix @djolly-fuze! :+1:

**Review by arm4b — APPROVED** [review]

**Pull request merged**